spring-cloud-gcp-data-datastore/src/main/java/com/google/cloud/spring/data/datastore/core/mapping/DatastoreMappingContext.java

```diff
@@ -55,6 +57,10 @@
 	// Kind and that are subclasses of the given class.
 	private static final Map<Class, Set<Class>> discriminationFamilies = new ConcurrentHashMap<>();
 
+	public DatastoreMappingContext() {
+		this.setSimpleTypeHolder(DatastoreNativeTypes.HOLDER);
```

Comment:
It would nice to have a test validating that the `simpleTypeHolder` was set to `DatastoreNativeTypes.HOLDER`.

Reply:
It would be nice to validate that both, built-in types and datastore native types got added... But it might not be possible, since `simpleTypeHolder` is not really exposed from `AbstractMappingContext`.

A method `shouldCreatePersistentEntityFor()` might be helpful, but that's protected, so you would have to subclass it to test.